### ISSUE TITLE
acquireTokenPopup redirects to about:blank#blocked in Chrome

### ISSUE THREAD

**Issue by jetilton** [AzureAD/microsoft-authentication-library-for-js#5574]
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

2.32.1

### Wrapper Library

Not Applicable

### Wrapper Library Version

None

### Public or Confidential Client?

Public

### Description

I have a Sveltekit app running on an Azure Static Web App.  I am trying to implement login with msal-browser.  I have taken a lot from the [msal-browser Typescript example](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples/TypescriptTestApp2.0) and everything works well in my development environment. When I push to the cloud the `acquireTokenPopup` opens up a blank popup and redirects to about:blank#blocked in chrome.  When I refresh and try again it only redirects to about:blank#blocked, it does not open a popup.  I am able to login with firefox, however.

The redirect happens so quickly I cannot see the error message and I was not able to find an error in the Chrome log file.  I am allowing all popups and cookies from third parties. 

### Error Message

Redirects before I can gather error

### MSAL Configuration

```javascript
auth: {
        clientId: "clientid",
        authority:"https://login.microsoftonline.com/consumers/"
    },
    cache: {
        cacheLocation: "sessionStorage", // This configures where your cache will be stored
        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
    },
    system: {
      loggerOptions: {
            loggerCallback: (level, message, containsPii) => {
                if (containsPii) {	
                    return;	
                }	
                switch (level) {	
                    case LogLevel.Error:	
                        console.error(message);	
                        return;	
                    case LogLevel.Info:	
                        console.info(message);	
                        return;	
                    case LogLevel.Verbose:	
                        console.debug(message);	
                        return;	
                    case LogLevel.Warning:	
                        console.warn(message);	
                        return;	
                }
            }
        }
    }
};
```

### Relevant Code Snippets

```javascript
import { PublicClientApplication, LogLevel} from "@azure/msal-browser";
import type {SilentRequest, AuthenticationResult, Configuration, AccountInfo, PopupRequest, EndSessionRequest} from "@azure/msal-browser";

/**
 * Configuration class for @azure/msal-browser: 
 * https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-browser/modules/_src_config_configuration_.html
 */
const MSAL_CONFIG: Configuration = {
    auth: {
        clientId: "03ad6ad7-32dc-470d-81ba-e1e78eb89a32",
        authority:"https://login.microsoftonline.com/consumers/"
    },
    cache: {
        cacheLocation: "sessionStorage", // This configures where your cache will be stored
        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
    },
    system: {
      loggerOptions: {
            loggerCallback: (level, message, containsPii) => {
                if (containsPii) {	
                    return;	
                }	
                switch (level) {	
                    case LogLevel.Error:	
                        console.error(message);	
                        return;	
                    case LogLevel.Info:	
                        console.info(message);	
                        return;	
                    case LogLevel.Verbose:	
                        console.debug(message);	
                        return;	
                    case LogLevel.Warning:	
                        console.warn(message);	
                        return;	
                }
            }
        }
    }
};

/**
 * AuthModule for application - handles authentication in app.
 */
export class AuthModule {

    private client: PublicClientApplication; // https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-browser/classes/_src_app_publicclientapplication_.publicclientapplication.html
    private account: AccountInfo | null; // https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-common/modules/_src_account_accountinfo_.html
    private loginSilent: SilentRequest;
    private loginPopup: PopupRequest; // https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-browser/modules/_src_request_popuprequest_.html
    private silentCustom: SilentRequest;
    private popupCustom: PopupRequest;

    constructor() {
        this.client = new PublicClientApplication(MSAL_CONFIG);
        this.account = null;
        this.loginPopup = {
            scopes: ["openid", "profile", "User.Read"]
        };
        this.loginSilent = {
            ...this.loginPopup
        }


    }

    private async getTokenSilent(silentRequest:SilentRequest):Promise<string|null>{
        let accounts = this.client.getAllAccounts()
        if(accounts.length){
            silentRequest.account = accounts[0]
            const response: AuthenticationResult = await this.client.acquireTokenSilent(silentRequest);
            return response.accessToken;
        }
        throw new Error('No accounts')

    }
    private async getTokenPopup(popupRequest:PopupRequest):Promise<string|null>{
        return this.client.acquireTokenPopup(popupRequest).then((resp) => {
            return resp.accessToken;
        }).catch((err) => {
            console.error(err);
            return null;
        });
    }


    private async getToken(silentRequest:SilentRequest, popupRequest:PopupRequest): Promise<string|null>{
        
        try{
            return await this.getTokenSilent(silentRequest)
        } catch(e){
            console.log("silent token acquisition fails. " + e);
        }try{
            return await this.getTokenPopup(popupRequest)
        }catch(e){
            console.log("popup token acquisition fails. " + e);
        }
        return null
    }


    async getLoginToken(): Promise<string|null>{
        return this.getToken(this.loginSilent, this.loginPopup)
    }


}



```
```


### Reproduction Steps

1. Go to the following [link](https://yellow-glacier-0da21f510.2.azurestaticapps.net/)
2. Select the login button on the upper right hand side


### Expected Behavior

I expect to be able to login with a popup and receive a token.

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

Chrome
Microsoft Edge

### Regression

_No response_

### Source

External (Customer)

**Comment by hectormmg** [comment]
@jetilton this seems like a Chrome popup configuration issue, please take a closer look at your browser settings.

**Comment by jetilton** [comment]
@hectormmg I am not sure what more I can look at on the Chrome side.  I have gone to settings and set my default behavior to "Sites can send pop-ups and use redirects".  I have also manually added my site to the "Allowed to send pop-ups and use redirects," exceptions list.  I have tried this from multiple computers.  Interestingly, Firefox is set to block pop-ups, but the login process runs smoothly.  Is there something in the Chrome settings you can suggest apart from the popup blocker?

**Comment by hectormmg** [comment]
@jetilton I still believe something in your config is most likely the issue, but you can try to set the `asyncPopups` flag to true in the `system` configuration options and see if this helps as a workaround. It will prevent "about:blank" from being loaded in the popup in the first place.

**Comment by sameerag** [comment]
@jetilton Please raise a new issue or link to this if the workaround @hectormmg suggested did not help unblock you. Closing this.
